Jenkinsfile-datastax

```diff
@@ -292,14 +288,6 @@ pipeline {
                           <th align="left">Choice</th>
                           <th align="left">Description</th>
                         </tr>
-                        <tr>
-                          <td><strong>2.1</strong></td>
-                          <td>Apache Cassandra&reg; v2.1.x</td>
-                        </tr>
-                        <tr>
-                          <td><strong>2.2</strong></td>
-                          <td>Apache Cassandra&reg; v2.2.x</td>
-                        </tr>
```

Comment:
According to the driver readme we should also be supporting these as well but (a) they're also EOL (and thus prolly excluded based on the logic above) and (b) the ASF CI is canonical for these versions.